src/Table.php

```diff
@@ -384,9 +405,19 @@ public function getRowTemplate()
     {
         $output = [];
         foreach ($this->columns as $name => $column) {
-            $field = $this->model->getElement($name);
 
-            $output[] = $column->getCellTemplate($field);
+            // If multiple formatters are defined, use the first for the header cell
```

Comment:
I'm unclear how to implement that. I need more info in #92 and #67. The mechanism is there, so I propose we just look for bugs and various combinations then address them.